app/assets/javascripts/cable.js

```diff
@@ -10,4 +10,4 @@
 
   App.cable = ActionCable.createConsumer();
 
-}).call(this);
+}).call(this);
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Newline required at end of file but not found                          eol-last